features/product/comparing_products/comparing_products.feature

```diff
@@ -0,0 +1,35 @@
+@comparing_products
+Feature: Adding product to the comparison
+    In order to make more reasonable choice of product
+    As a Visitor
+    I want to see comparison of products
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Sylius Mug"
+        And the store has a product "Sylius Cup"
+        And the store has a product "Sylius T-Shirt"
+
+    @ui
+    Scenario: comparing two products with common attributes
+        Given There is a product "Sylius Mug" with "Color" attribute set to "Red"
```

Comment:
Shouldn't this step be like...
`Given the "Sylius Mug" product has the "Color" attribute set to "Red"`

because your product is already defined/know in this scenario, as you create it in the `Background:` part :)

Reply:
and btw. `There` -> `there`. These are natural sentences, so only the first word (`Given`) should be capitalized :D